packages/api/endpoints/granules.js

```diff
@@ -206,7 +248,6 @@ async function put(req, res) {
       status: 'SUCCESS',
     });
   }
-
   return res.boom.badRequest('Action is not supported. Choices are "applyWorkflow", "move", "reingest", or "removeFromCmr"');
```

Comment:
This feels like it might be incorrect now, given <none> is an option.  

I grok how/why we got to specifying no action in the body results in an update, but perhaps something like would be more correct: 

```suggestion
  return res.boom.badRequest('Action is not supported. Choices are "applyWorkflow", "move", "reingest", "removeFromCmr" or specify no "action" to update an existing granule');
```

Reply:
It's admittedly a use-edge case, but users can and do things like put 'update' in the action field after misreading docs.     This shouldn't hold up the PR, it's a take/leave suggestion only.

Reply:
Ok, I can see where you're coming from.